scheduler-worker/build.gradle

```
@@ -124,6 +124,7 @@ dependencies {
   testImplementation project(':examples:foo-missionmodel')
   testImplementation testFixtures(project(':scheduler-server'))
   testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.0'
+  testImplementation 'org.junit.jupiter:junit-jupiter-params:5.10.0'
   testImplementation 'org.assertj:assertj-core:3.24.2'
   testImplementation 'junit:junit:4.13.2'
```

Minor, can ignore (might not even belong in this PR), but we actually don't need to be importing both JUnit 4 and JUnit 5.
```suggestion
  testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.0'
  testImplementation 'org.junit.jupiter:junit-jupiter-params:5.10.0'
  testImplementation 'org.assertj:assertj-core:3.24.2'
```

---

I have removed this dependency as the last commit.